### PR TITLE
System: update Chart.js library to v3.6.0 and migrate existing charts

### PR DESCRIPTION
**Pull request by SKuipers**
Several of the charts used in Gibbon were not displaying well on mobile devices. This PR updates the chart.js library to the latest version and makes any required updates to chart options based on the recent version. It also removes old versions of the chart.js library.

**Motivation and Context**

**How Has This Been Tested?**
Locally.

### CODE REVIEW

File: modules/Rubrics/src/Visualise.php [hunk 1]
```diff
@@ -155,6 +158,13 @@ public function renderVisualise($legend = true, $image = false, $path = '', $id
                         this.options.animation.onComplete = null;
                     }'),
                 ];
+            } else {
```
Comment (rossdotparker) [inline modules/Rubrics/src/Visualise.php:161]: Interested to know why the commented section has been left in here. I know there will be a good reason!

File: modules/Staff/report_absences_weekly.php [hunk 2]
```diff
@@ -84,21 +84,22 @@
 
     // BAR GRAPH
     $chartConfig = [
-        'height' => '70',
+        'height' => '30vh',
         'tooltips' => [
```
Comment (rossdotparker) [inline modules/Staff/report_absences_weekly.php:88]: Should this one be singular?

Comment (SKuipers) [reply modules/Staff/report_absences_weekly.php:88]: Yes it should! Well caught.

File: modules/Rubrics/src/Visualise.php (hunk 1 above)
Comment (SKuipers) [reply modules/Rubrics/src/Visualise.php]: Oops! Nope, no good reason this time 😆

**Pull request merged**